bundle/src/main/java/com/cognifide/cq/cqsm/core/progress/ProgressHelper.java

```diff
@@ -45,4 +46,16 @@ public static String toJson(List<ProgressEntry> entries) {
 	public static String toJson(ProgressEntry entry) {
 		return GSON.toJson(entry);
 	}
+
+	public static boolean calculateSuccess(List<ProgressEntry> entries) {

```

Comment:
I know that it was calculateSuccess earlier, but if You have moved it already, then we could change this nape to be more meaningful, like `hasErrors` or something like that.

Reply:
I renamed it to `hasNoErrors` method.